### PR TITLE
Update text read-the-docs

### PR DESCRIPTION
**Pull request by Rensvandeschoot**

### CODE REVIEW

File: docs/source/intro/vocabulary.rst [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Vocabulary
 ----------
 
-When we write about ASReview, we want to use a consistent vocabulary.
+When writing about ASReview, please use the following vocabulary:
```
Comment (J535D165) [inline docs/source/intro/vocabulary.rst:4]:
```suggestion
The ASReview project makes use of standardized terminology for all communication regarding ASReview and its underlying technology. An overview of terms and usage can be found in the table below.  
```

Avoid Please https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please

File: docs/source/intro/vocabulary.rst [hunk 2]
```diff
@@ -37,19 +42,19 @@ When we write about ASReview, we want to use a consistent vocabulary.
...
+|                  | The records can contain both information that is used for |
 |                  | training the active learning model, and information that  |
 |                  | is not used for this purpose.                             |
 |                  |                                                           |
-|                  | In the case of systematic reviews, a record is a          |
-|                  | scientific publication. Here, the information that is     |
-|                  | used for training purposes is the text in the title and   | 
+|                  | In the case of systematic reviewing, a record is meta-data|
+|                  | for a scientific publication. Here, the information that  |
+|                  | is used for training purposes is the text in the title and|
```
Comment (J535D165) [inline docs/source/intro/vocabulary.rst:57]: Was this tested? I'm not sure whether space before the table boundary is required.

**Review by J535D165 — COMMENTED** [review]
Thanks

File: docs/source/intro/vocabulary.rst (hunk 2 above)
Comment (Rensvandeschoot) [reply docs/source/intro/vocabulary.rst:57]: I have not tested it, sorry

**Pull request merged**